src/js/plugins/menu_toggle/plugin.menu_toggle.js

```diff
@@ -1,113 +1,194 @@
 
-/* global: $ */
+/* global: $, BookReader */
 /**
- * Plugin for managing menu in full screen
- * Enabling this plug-in removes the "menu tab" triangle
- * and uses center touch/click to show/hide the menu below
- * Behavior is predicated on custom event: `brFullScreenToggled`
- * This is fired when user clicks on the fullscreen button on the menu
+ * Plugin for managing menu visibility
+ * Enabling this plug-in:
+ * + removes the "menu tab" triangle
+ * + toggles nav at: book center tap/click
+ * + toggles nav at: black background tap/click
  *
- * This uses BookReader native functions and parameters to check its UI state.
- * This includes:
+ * This uses core BookReader functions and parameters to check its UI state:
  * - br.refs = (at best) ui references that are present at any given time
  * - br.navigationIsVisible() - checks using refs to confirm the navbar's presence
+ * - br.showNavigation() & br.hideNavigation()
  *
+ * The list of BookReader custom events this plugin taps into are mainly
+ * listed in the `.init` function
  */
 
 (function addMenuToggler() {
     jQuery.extend(BookReader.defaultOptions, {
       enableMenuToggle: true
     });
 
+    /**
+     * Hides Nav arrow tab
+     *
+     * @param { object } br - BookReader instance
+     */
     function hideArrow(br) {
       if (!br.refs || !br.refs.$BRnav) {
         return;
       }
-      var $menuTab = br.refs.$BRnav.children('.js-tooltip');
+      var $menuTab = br.refs.$BRnav.children('.BRnavCntl');
       $menuTab.css('display', 'none');
     }
 
+    /**
+     * Sets up nav - hides arrow tab & adds click events
+     *
+     * @param { object } br - BookReader instance
+     */
     function setupNavForToggle(br) {
       hideArrow(br);
-      registerEventHandlers(br);
+      registerClickHandlers(br);
     }
 
+    /**
+     * Resets nav to always show
+     * hides arrow tab, removes click events, shows nav chrome
+     *
+     * @param { object } br - BookReader instance
+     */
     function alwaysShowNav(br) {
       hideArrow(br);
-      removeToggleFromNav();
-      br.showNavigation();
-    }
-  
-    var removeToggleFromNav = function removeToggleFromNav(br) {
-      if (!br.refs || !br.refs.$BRnav) {
-        return;
-      }
-      var $menuTab = br.refs.$BRnav.children('.js-tooltip');
-      $menuTab.css('display', 'block');
       removeEventHandlers(br);
+      br.showNavigation();
     }
   
+    /**
+     * Removes click handlers on elements that house the book pages
+     *
+     * @param { object } br - BookReader instance
+     */
     var removeEventHandlers = function removeEventHandlers(br) {
```

Comment:
Is this method the opposite of the `registerClickHandlers` function that was renamed? If so I might rename this to `removeClickHandlers` for parallel naming structure.

Reply:
thanks for catching